ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -79,6 +79,7 @@ conformance_test_tool_args = [
         "TLSAtLeastOnePointTwoIT",
         "CommandDeduplicationPeriodValidationIT:OffsetPruned",  # requires pruning not available in canton community
         "DeeplyNestedValueIT",  # FIXME: Too deeply nested values flake with a time out (half of the time)
+        "InterfaceSubscriptionsIT:ISTransactionsUnknownTemplateOrInterface",  # FIXME: Remove exclusion after the next Canton SDK upgrade
```

Comment:
ah, this doesn't work since the test is not packaged in lf 1.14 test tool so the exclusion fails 😭

Reply:
Can't you add it on line 130 below? This should be needed only for dev, no?

Reply:
I though so as well, but the `lf_versions` of the `conformance-test-dev-domain` include default as well (which is 1.14 afaik).